Amigolden/src/app/components/map/map.component.ts

```diff
@@ -34,51 +44,59 @@ export class MapComponent implements OnInit {
   zoom: number;
   address: string;
   isCreating = false;
-  locationEntitiesMap = new Array<{location: Location, data: Array<any>}>();
+
+  locationEntitiesMap = new Array<{ location: Location; data: Array<any> }>();
   private geoCoder;
+  searchForm: FormGroup;
 
-  @ViewChild('search', {static: false })
+  @ViewChild("search", { static: false })
   public searchElementRef: ElementRef;
 
-
   constructor(
     private mapsAPILoader: MapsAPILoader,
     private ngZone: NgZone,
-    protected modalController: ModalController
-  ) { }
+    protected modalController: ModalController,
+    private fb: FormBuilder
+  ) {}
 
   resolveMapData() {
-    this.options.getData(this.latitude, this.longitude, this.withinMiles).subscribe(entities => {
-      const entitiesMap = entities.map(e => ({ location: this.options.locationResolver(e), data: e}));
-      const group = (array, prop) => array.reduce((g, item) => {
-        const propVal = item[prop];
-        const identifier = propVal.id;
-        const entry = g[identifier];
-        const data = item.data;
-
-        if (entry) {
-          entry.data.push(data);
-        } else {
-          g[identifier] = {location: propVal, data: [data]};
-        }
-        return g;
-      }, {});
-      const groups = group(entitiesMap, 'location');
-      this.locationEntitiesMap = Object.keys(groups).map((key) => groups[key])
-            || new Array<{location: Location, data: Array<any>}>();
-    });
+    this.options
+      .getData(this.latitude, this.longitude, this.withinMiles)
+      .subscribe((entities) => {
+        const entitiesMap = entities.map((e) => ({
+          location: this.options.locationResolver(e),
+          data: e,
+        }));
+        const group = (array, prop) =>
+          array.reduce((g, item) => {
+            const propVal = item[prop];
+            const identifier = propVal.id;
+            const entry = g[identifier];
+            const data = item.data;
+
+            if (entry) {
+              entry.data.push(data);
+            } else {
+              g[identifier] = { location: propVal, data: [data] };
+            }
+            return g;
+          }, {});
+        const groups = group(entitiesMap, "location");
+        this.locationEntitiesMap =
+          Object.keys(groups).map((key) => groups[key]) ||
+          new Array<{ location: Location; data: Array<any> }>();
+      });
   }
 
   closeModal() {
     this.modalController.dismiss();
   }
 
   selectLocation(location: Location, data: any) {
-    this.locationSelected.emit({location, data});
+    this.locationSelected.emit({ location, data });
```

Comment:
fix white space changes,  general rule of thumb, if its not related to the task don't commit it. It makes it harder to view what is required to be changed vs whats just whitespace etc